<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>luksdeletekey</class>
 <widget class="QDialog" name="luksdeletekey">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>remove a key from a volume</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLineEdit" name="lineEditPassphrase">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassphrase">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>key </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="rbPassphrase">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>existing key in the volume to delete</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPassphraseFromFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>existing key from a key file to delete</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonOpenKeyFile">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>open a keyfile</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;delete</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;cancel</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVolumePath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>volume path</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOpenVolume">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>open an encrypted file</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOpenPartition">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>open an encrypted partition</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditVolumePath</tabstop>
  <tabstop>pushButtonOpenVolume</tabstop>
  <tabstop>pushButtonOpenPartition</tabstop>
  <tabstop>lineEditPassphrase</tabstop>
  <tabstop>pushButtonOpenKeyFile</tabstop>
  <tabstop>rbPassphrase</tabstop>
  <tabstop>rbPassphraseFromFile</tabstop>
  <tabstop>pushButtonDelete</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
